### PR TITLE
4240 login messages

### PR DESCRIPTION
**Pull request by dlangston**
@DanielJMaher I didn't add a Success message because the modal disappears before you can read one. I can add a pop up but that will just create one more step to logging in. Plus, once you are logged in your user name is displayed on the navbar.

screen shot 2015-09-30 at 3 03 56 pm
screen shot 2015-09-30 at 3 04 01 pm
screen shot 2015-09-30 at 3 04 06 pm

### CODE REVIEW

**Comment by FBRTMaka** [comment]
@dlangston , is this good to go?

**Comment by dlangston** [comment]
should be if it looks okay to you

**Comment by DanielJMaher** [comment]
Wait.. what happens after the three attempts?

**Comment by DanielJMaher** [comment]
@dlangston

**Comment by DanielJMaher** [comment]
@FBRTMaka Hold off for a second

**Comment by dlangston** [comment]
@DanielJMaher 
you are locked out until you close and reopen login

**Comment by DanielJMaher** [comment]
@dlangston Let's not give them a limit.  I like the idea for now, but lets just give them an infinite amount for now and jsut tel them the combo is wrong. Don't delete the cod ethough, we might want it later when password reset works.

**Comment by dlangston** [comment]
Sounds good.

**Comment by dlangston** [comment]
@DanielJMaher made the message blink every time you enter the wrong password. i left in the counter sense i had already done it. let me know if you want me to remove it though.

screen shot 2015-10-01 at 10 35 16 am

**Comment by dlangston** [comment]
@DanielJMaher removed the counter 
screen shot 2015-10-01 at 10 44 45 am

**Comment by DanielJMaher** [comment]
@birdage

File: ooiui/static/js/views/common/LoginView.js [hunk 1]
```diff
@@ -49,6 +50,7 @@ var LoginView = Backbone.View.extend({
     }
     this.attempts = 0; // Keep track of attempts
     this.isHidden = false; // Initially keep retrying
+    this.attemptsTotal = 3
```
Comment (FBRTMaka) [inline ooiui/static/js/views/common/LoginView.js]: semicolon

**Comment by dlangston** [comment]
@FBRTMaka 
added ;

**Comment by FBRTMaka** [comment]
@dlangston Yay! Looks great! Merging!

**Pull request merged**